nixos/modules/programs/firejail.nix

```diff
@@ -20,13 +19,38 @@ let
     '';
   };
 
+  wrappedPkgs = map (pkg:
+    pkgs.stdenv.mkDerivation {
+      name = "firejail-" + pkg.name;
+      buildCommand = ''
+        mkdir -p $out
+        ln -s ${pkg}/* $out
+        rm $out/bin
+        mkdir $out/bin
+        for bin in ${pkg}/bin/*; do
```

Comment:
I think the variable name `bin` is confusing here. Maybe `dir` instead?

Reply:
These are actual extecutables.

Reply:
Oh nevermind, I don't know why I thought they were subdirectories.